components/Context/index.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { ChangeEvent, useCallback, useEffect, useState} from "react";
+import UrlButton from "./UrlButton";
+import { Card, ICard } from "./Card";
+import { clearIndex, crawlDocument } from "./utils";
+
+import { Button } from "./Button";
+interface ContextProps {
+  className: string;
+  selected: string[] | null;
+  uploads: Array<string>
+}
+
+export const Context: React.FC<ContextProps> = ({ className, selected, uploads }) => {
+  /*const [entries, setEntries] = useState([]);*/
+  console.log("UPLOADS ", uploads);
+  const [cards, setCards] = useState<ICard[]>([]);
+
+  const [splittingMethod, setSplittingMethod] = useState("markdown");
+  const [chunkSize, setChunkSize] = useState(256);
+  const [overlap, setOverlap] = useState(1);
+
+  // Scroll to selected card
```

Comment:
delete this code since it is unused?

Reply:
I think it'll be used for the scrollbar/chunk size selector? just hidden rn